code/game/objects/items/food/cheese.dm

```diff
@@ -95,7 +95,7 @@
 	rat_heal = 35
 
 /obj/item/food/cheese/firm_cheese/MakeProcessable()
-	AddElement(/datum/element/processable, TOOL_KNIFE, /obj/item/food/cheese/firm_cheese_slice, 3, 3 SECONDS)
+	AddElement(/datum/element/processable, TOOL_KNIFE, /obj/item/food/cheese/firm_cheese_slice, 3, 3 SECONDS, screentip_verb = "Slice")
```

Comment:
Why not add this to the cheese subtype?

Reply:
having trouble with this one, going to investigate further but i finished the other suggestions.

just having the var/food/cheese/slice_type doesnt work and /food/cheese_slice isnt a subclass and I dont want to introduce it as one on account of all of the files i'd have to touch, so ill try to figure out why that doesnt work and see if theres something I can do that doesnt involve reclassing cheese.

Reply:
I see what you mean, slice is the same subtype. You can leave this as is, unless you feel like reworking the cheese types

Reply:
i would consider it if there was more sliceable cheese but given all of the files that /cheese/wedge touches i think it would be better to just let this file have the two proc calls, it would be a lot of files that have to be changed to have one less proc call